test/integration/targets/wait_for/tasks/main.yml

```diff
@@ -98,7 +121,7 @@
       - "waitfor.msg == 'fail with custom message'"
 
 - name: setup start SimpleHTTPServer
-  shell: sleep 10 && cd {{ files_dir }} && {{ ansible_python.executable }} {{ output_dir}}/testserver.py {{ http_port }}
+  shell: sleep 3 && cd {{ files_dir }} && {{ ansible_python.executable }} {{ output_dir}}/testserver.py {{ http_port }}
```

Comment:
Why change sleep here?  It seems like that might have been tuned for timing in CI or similar.  If it's just to take less time maybe we should change it in a separate PR.

Reply:
And the same comment for all the other timing changes.